packages/@react-aria/overlays/src/useOverlayPosition.ts

```diff
@@ -58,7 +58,7 @@ export interface AriaPositionProps extends PositionProps {
    * The minimum distance the arrow's edge should be from the edge of the overlay element.
    * @default 0
    */
-  minOverlayArrowOffset?: number
+  arrowBoundaryOffset?: number
```

Comment:
Not sure if we need to update the doc comments at all

Reply:
Seems like a fine description to me, i vote for keeping as is

Reply:
I considered updating the comment for the `arrowSize` but decided against it since other libraries (floatingUI) use the same kind of "cross" vocab